app/models/harvest.rb

```diff
@@ -7,6 +7,7 @@ class Harvest < ActiveRecord::Base
   belongs_to :crop
```

Comment:
Do we have any guarantee that `harvest.crop == harvest.planting.crop` where both are defined?